idd/Energy+.idd.in

```diff
@@ -50198,6 +50202,7 @@ HeatExchanger:AirToAir:FlatPlate,
       \type real
       \minimum> 0.0
       \units m3/s
+      \autosizable
```

Comment:
I don't know what you two are talking about.  All of both of your comments accepted and the PR will be updated.